docs/pages/server-access/guides/ssh-pam.mdx

```diff
@@ -124,84 +100,67 @@ ssh_service:
     # "false" by default
     use_pam_auth: true
     # sets custom environment variables for PAM modules
-    # no environment variables except the standard `TELEPORT_USERNAME`, `TELEPORT_LOGIN`, and `TELEPORT_ROLES`
     environment:
       FOO: "bar"
       EMAIL: "{{ external.email }}"
 ```
 
-### Example node with PAM turned off
+Teleport can also read PAM environment variables from the PAM **handle**, an
+opaque data structure that is used by PAM to store state. These variables
+include:
 
-```yaml
-teleport:
-  nodename: graviton-node-1
-  auth_token: hello
-  auth_servers:
-  - 10.2.1.230:5070
-  data_dir: /var/lib/teleport
-proxy_service:
-  enabled: false
-auth_service:
-  enabled: false
-ssh_service:
-  enabled: true
-  # configures PAM integration. see below for more details.
-  pam:
-     enabled: false
-```
+- `TELEPORT_USERNAME`: The Teleport username of the user who is logging into a Node. This is usually an email address (such as `user@example.com`) if using SAML/OIDC identities with Teleport Enterprise, or a more standard `exampleuser` if using local Teleport users.
+- `TELEPORT_LOGIN`: The name of the Linux/Unix principal that the Teleport user is logging into the Teleport Node, e.g., `root`, `developer`, `ubuntu`, `ec2-user`,or similar.
```

Comment:
`... is logging into the Teleport Node **with** ...`

Also not sure on the phrasing "Linux/Unix principal".  Maybe just "Linux/Unix user"?